src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/DiagController.cs

```diff
@@ -564,8 +564,9 @@ public ActionResult<DotnetMonitorInfo> GetInfo()
         }
 
         [HttpPost("parameters", Name = nameof(CaptureParameters))]
-        [ProducesWithProblemDetails(ContentTypes.ApplicationJson)]
+        [ProducesWithProblemDetails(ContentTypes.ApplicationNdJson, ContentTypes.ApplicationJson, ContentTypes.TextPlain)]
```

Comment:
Similar to my earlier comment, we shouldn't offer standard json here, only `ContentTypes.ApplicationNdJson`, `ContentTypes.ApplicationJsonSequence`, and `ContentTypes.TextPlain` to be consistent with our other APIs.